### PR TITLE
firefly-632:More issues found after fixing FIREFLY-629 and 631

### PR DESCRIPTION
**Pull request by lznakano**
https://jira.ipac.caltech.edu/browse/FIREFLY-632

 This PR fixed some of the issues reported in the ticket above.  

NOTE the following two issues:
GRID:  FF can not add grid for the image without projection, ie. the mapType = unrecognized or unspecified.  I spent quit some effort on this issue, but I did not find a better solution.  
GREAT LEVEL-1: Some of the GREAT LEVEL-1 data has 'Table Load" error. The cause is the FF does not support the FITs that has more than one naxisi>1 where i=1,..naxis.  The failing FITs files have both naxis1>1 and naxis2>1.

Test URL:
https://lijun.irsakudev.ipac.caltech.edu/applications/sofia/
https://lijun1.irsakudev.ipac.caltech.edu/applications/sofia/

To test:
1. allSky
2. Select instrument: GREAT, LEVEL_1

In the first URL, the fourth row will give a "table load" error.
In the second URL, the bug was fixed, it just shows the image.

### CODE REVIEW

File: src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java [hunk 1]
```diff
@@ -78,21 +78,36 @@ public FileAnalysisReport analyzeFits(FileAnalysisReport inputReport,
...
+              return setReportDisplay(inputReport, FileAnalysisReport.ChartTableDefOption.showChart);
             }
         }
 
         return inputReport;
     }
 
+    private FileAnalysisReport setReportDisplay(FileAnalysisReport report,FileAnalysisReport.ChartTableDefOption option ){
+
+        FileAnalysisReport retReport =  report.copy();
```
Comment (ejoliet) [inline src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java:103]: I'm not sure that you even need to copy the report to set the parts with the default showing.

Comment (lznakano) [reply src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java:103]: I saw you used the copy and just kept the same manner.  I don't think that we need to make copy any places at all.  If you are OK, I will remove all the copy and update the input directly.  Again, it is your call.

Comment (ejoliet) [reply src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java:103]: Ok, please,  if the copy is not needed, we should remove it, and test everything again before merging. Thank you.

Comment (lznakano) [reply src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java]: > Ok, please, if the copy is not needed, we should remove it, and test everything again before merging. Thank you.

I did quite some clean up.   Removed the unnecessary copy's and re-arranged the codes.  I tested and seemed all OK.

**Pull request merged**